### PR TITLE
Move index.html into docs/.

### LINKED ISSUE #83: Move index.html into docs
@jakubpawlowicz I'll make a PR but you will need to change the repo setting to use the master/docs folder.

Then I will add a .nojekyll file and this way we separate code and docs. The default branch should be master then, and gh-pages can be deleted.

I'll make a PR and ping you to make the changes

### PR DESCRIPTION
**Pull request by XhmikosR**
Also, add a .nojekyll file so that the file isn't processed by Jekyll at all.

@jakubpawlowicz this is the patch. Please go to the repo settings and change GH Pages to deploy from master/docs. After this is merged, we need to make master the default branch and delete the gh-pages branch. Alternatively, you can give me admin rights :)

Fixes #83

### CODE REVIEW

**Comment by jakubpawlowicz** [comment]
@XhmikosR makes sense to me although I wouldn't mind more than one commit for those changes. And you already have write rights in this repo, don't you?

**Comment by XhmikosR** [comment]
This needs changes in the repo settings and branches.

**Comment by jakubpawlowicz** [comment]
Gotcha. Would you mind splitting the commits before I merge it?

**Comment by XhmikosR** [comment]
There's nothing to split. It's just one patch and then you need to follow
the steps I mentioned above.

File: README.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ Run `node ./bin/bench --gzip` to measure the gzip size instead of the regular fi
 
 ## License
 
-css-minification-benchmark is released under the [MIT License](https://github.com/GoalSmashers/clean-css/blob/master/LICENSE).
+css-minification-benchmark is released under the [MIT License](https://github.com/GoalSmashers/css-minification-benchmark/blob/master/LICENSE).
```
Comment (jakubpawlowicz) [inline README.md]: Well, this is unrelated to the PR.

**Comment by XhmikosR** [comment]
Yeah but I'm too lazy to split it.

**Comment by XhmikosR** [comment]
@jakubpawlowicz can we move with this please? It makes a lot more sense to get rid of the gh-pages branch. I can't set it up myself since I don't have admin rights in the repo.

**Comment by jakubpawlowicz** [comment]
Fair and square.

**Comment by XhmikosR** [comment]
@jakubpawlowicz please delete the gh-pages branch, set master as the default branch and set GitHub pages to deploy from master/docs. Otherwise merging this is moot 😛

**Comment by jakubpawlowicz** [comment]
Done.

**Comment by XhmikosR** [comment]
@jakubpawlowicz Can you also change the default branch to master and delete the gh-pages branch?
